gradle.properties

```diff
@@ -34,6 +34,7 @@ APTOIDE_WEB_SERVICES_WRITE_V7_HOST_DEVELOPMENT=dev.ws75-primary.aptoide.com
 APTOIDE_WEB_SERVICES_APTWORDS_HOST_DEVELOPMENT=dev.aptoide.com
 APTOIDE_WEB_SERVICES_SEARCH_HOST_DEVELOPMENT=buzz.aptoide.com:10001
 APTOIDE_WEB_SERVICES_AB_TESTING_HOST=abtesting.aptoide.com
+APTOIDE_WEB_SERVICES_APICHAIN_BDS_HOST=0
```

Comment:
Is this API supposed to be private or public? All other are public here. Because if they are supposed to be private we need to handle this or this will break somewhere.